truenas/webui
# NAS-111705 / 21.08 / NAS-111705 - Better reloading of directory services page when AD/LDAP…

Testing:
1. Login to cloud VM
2. Enable/disable active directory on Directory Services page.
3. Make sure the directory services page properly updates.
